.gitignore

```diff
@@ -1,3 +1,6 @@
 *~
 docs/build
 Manifest.toml
+src/Healpix.jl
+src/sphtfunc.jl
+test/test_sphtfunc.jl
```

Comment:
Why were these files added to `.gitignore`?

Reply:
I was worried of pushing wrong changes while working on the other functions for the next PR. It was just temporary because I then created new files locally for that. I was sure I re-changed the `.gitignore` but I probably forgot to push it. It should be fixed now.